### PR TITLE
updated redir

### PR DESCRIPTION
**Pull request by joel-hamill**

### CODE REVIEW

**Comment by joel-hamill** [comment]
this should replace https://github.com/dcos/dcos-website/pull/730

File: redirects [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+/documentation /docs/1.7/
+/docs/1.7/getting-started/overview/index.html /docs/1.7/usage/
 /media-kit https://mesosphere.invisionapp.com/boards/U61L5GZM5CYG3/
 /media-kit/index.html https://mesosphere.invisionapp.com/boards/U61L5GZM5CYG3/
-/overview/components/index.html /docs/latest/overview/components/
+/docs/1.7/overview/components/index.html /docs/1.7/overview/components/
 /support /community/
-/usage/cli/install/index.html /docs/latest/usage/cli/install/
-/docs/1.7/usage/tutorials/ruby-on-rails/index.html /docs/1.7/usage/tutorials/
+/docs/1.7/usage/cli/install/index.html /docs/1.7/usage/cli/install/
```
Comment (karlkfi) [inline redirects]: I don't think this is what you meant to do. The intent was to grab links that don't use the docs path prefix.

File: redirects [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+/docs/1.7 /docs/1.7/
 /docs/support/index.html /community/
-/documentation /docs/latest/
-/getting-started/overview/index.html /docs/latest/usage/
+/documentation /docs/1.7/
+/docs/1.7/getting-started/overview/index.html /docs/1.7/usage/
 /media-kit https://mesosphere.invisionapp.com/boards/U61L5GZM5CYG3/
 /media-kit/index.html https://mesosphere.invisionapp.com/boards/U61L5GZM5CYG3/
-/overview/components/index.html /docs/latest/overview/components/
+/docs/1.7/overview/components/index.html /docs/1.7/overview/components/
```
Comment (karlkfi) [inline redirects]: same problem here. meant to catch links that don't prefix with /docs

File: redirects [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
+/docs/1.7/advanced-installer/create-a-script-for-ip-address-discovery/index.html /docs/1.7/administration/installing/custom/advanced/
+/docs /docs/1.7/
+/docs/administration/installing/custom/automated-installer/system-requirements/index.html /docs/1.7/administration/installing/custom/system-requirements/
+/docs/index.html /docs/1.7/
+/docs/1.7 /docs/1.7/
 /docs/support/index.html /community/
-/documentation /docs/latest/
-/getting-started/overview/index.html /docs/latest/usage/
+/documentation /docs/1.7/
+/docs/1.7/getting-started/overview/index.html /docs/1.7/usage/
```
Comment (karlkfi) [inline redirects]: same issue here

File: redirects [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
+/docs/1.7/administration/service-discovery/index.html /docs/1.7/overview/service-discovery/
+/docs/1.7/advanced-installer/create-a-script-for-ip-address-discovery/index.html /docs/1.7/administration/installing/custom/advanced/
+/docs /docs/1.7/
+/docs/administration/installing/custom/automated-installer/system-requirements/index.html /docs/1.7/administration/installing/custom/system-requirements/
+/docs/index.html /docs/1.7/
+/docs/1.7 /docs/1.7/
 /docs/support/index.html /community/
-/documentation /docs/latest/
-/getting-started/overview/index.html /docs/latest/usage/
+/documentation /docs/1.7/
```
Comment (karlkfi) [inline redirects]: this should go to /docs/latest where it will then be redirected again by the prefix redir

File: redirects [hunk 5]
```diff
@@ -1,29 +1,29 @@
...
+/docs/1.7/administration/installing/custom/gui/index.html /docs/1.7/administration/installing/custom/gui/
+/docs/1.7/administration/installing/index.html /docs/1.7/administration/installing/
+/docs/1.7/administration/opt-out/index.html /docs/1.7/administration/opt-out/
+/docs/1.7/administration/security-and-authentication/index.html /docs/1.7/overview/security/
+/docs/1.7/administration/service-discovery/index.html /docs/1.7/overview/service-discovery/
+/docs/1.7/advanced-installer/create-a-script-for-ip-address-discovery/index.html /docs/1.7/administration/installing/custom/advanced/
+/docs /docs/1.7/
+/docs/administration/installing/custom/automated-installer/system-requirements/index.html /docs/1.7/administration/installing/custom/system-requirements/
+/docs/index.html /docs/1.7/
+/docs/1.7 /docs/1.7/
```
Comment (karlkfi) [inline redirects]: revert this. it's for catching things that forgot the trailing slash.

File: redirects [hunk 6]
```diff
@@ -1,29 +1,29 @@
...
+/docs/1.7/administration/dcosarchitecture/security/index.html /docs/1.7/overview/security/
+/docs/1.7/administration/installing/custom/automated-installer/system-requirements/index.html /docs/1.7/administration/installing/custom/system-requirements/
+/docs/1.7/administration/installing/custom/cli/index.html /docs/1.7/administration/installing/custom/cli/
+/docs/1.7/administration/installing/custom/gui/index.html /docs/1.7/administration/installing/custom/gui/
+/docs/1.7/administration/installing/index.html /docs/1.7/administration/installing/
+/docs/1.7/administration/opt-out/index.html /docs/1.7/administration/opt-out/
+/docs/1.7/administration/security-and-authentication/index.html /docs/1.7/overview/security/
+/docs/1.7/administration/service-discovery/index.html /docs/1.7/overview/service-discovery/
+/docs/1.7/advanced-installer/create-a-script-for-ip-address-discovery/index.html /docs/1.7/administration/installing/custom/advanced/
+/docs /docs/1.7/
```
Comment (karlkfi) [inline redirects]: revert this. this is for catching unversioned /docs links without a trailing slash

File: redirects [hunk 7]
```diff
@@ -1,29 +1,29 @@
...
+/docs/administration/installing/custom/automated-installer/system-requirements/index.html /docs/1.7/administration/installing/custom/system-requirements/
+/docs/index.html /docs/1.7/
```
Comment (karlkfi) [inline redirects]: revert this too

File: redirects [hunk 8]
```diff
@@ -1,29 +1,31 @@
...
 /documentation /docs/latest/
 /getting-started/overview/index.html /docs/latest/usage/
 /media-kit https://mesosphere.invisionapp.com/boards/U61L5GZM5CYG3/
 /media-kit/index.html https://mesosphere.invisionapp.com/boards/U61L5GZM5CYG3/
-/overview/components/index.html /docs/latest/overview/components/
+/overview/components/index.html /overview/components/
 /support /community/
 /usage/cli/install/index.html /docs/latest/usage/cli/install/
 /docs/1.7/usage/tutorials/ruby-on-rails/index.html /docs/1.7/usage/tutorials/
+
```
Comment (karlkfi) [inline redirects]: don't need extra line breaks

File: redirects [hunk 9]
```diff
@@ -1,29 +1,31 @@
...
+/docs/1.7/administration/installing/custom/cli/index.html /docs/1.7/administration/installing/custom/cli/
+/docs/1.7/administration/installing/custom/gui/index.html /docs/1.7/administration/installing/custom/gui/
+/docs/1.7/administration/installing/index.html /docs/1.7/administration/installing/
+/docs/1.7/administration/opt-out/index.html /docs/1.7/administration/opt-out/
+/docs/1.7/administration/security-and-authentication/index.html /docs/1.7/overview/security/
+/docs/1.7/administration/service-discovery/index.html /docs/1.7/overview/service-discovery/
+/docs/1.7/advanced-installer/create-a-script-for-ip-address-discovery/index.html /docs/1.7/administration/installing/custom/advanced/
 /docs /docs/latest/
-/docs/administration/installing/custom/automated-installer/system-requirements/index.html /docs/latest/administration/installing/custom/system-requirements/
+/docs/administration/installing/custom/automated-installer/system-requirements/index.html /docs/1.7/administration/installing/custom/system-requirements/
```
Comment (karlkfi) [inline redirects]: revert this

File: redirects [hunk 10]
```diff
@@ -1,29 +1,31 @@
...
+/docs/1.7/usage/services/uninstall/#framework-cleaner /docs/1.7/usage/managing-services/uninstall/#framework-cleaner
+/docs/1.7/administration/dcosarchitecture/security/index.html /docs/1.7/overview/security/
+/docs/1.7/administration/installing/custom/automated-installer/system-requirements/index.html /docs/1.7/administration/installing/custom/system-requirements/
+/docs/1.7/administration/installing/custom/cli/index.html /docs/1.7/administration/installing/custom/cli/
+/docs/1.7/administration/installing/custom/gui/index.html /docs/1.7/administration/installing/custom/gui/
+/docs/1.7/administration/installing/index.html /docs/1.7/administration/installing/
+/docs/1.7/administration/opt-out/index.html /docs/1.7/administration/opt-out/
+/docs/1.7/administration/security-and-authentication/index.html /docs/1.7/overview/security/
+/docs/1.7/administration/service-discovery/index.html /docs/1.7/overview/service-discovery/
+/docs/1.7/advanced-installer/create-a-script-for-ip-address-discovery/index.html /docs/1.7/administration/installing/custom/advanced/
```
Comment (karlkfi) [inline redirects]: revert this

File: redirects [hunk 11]
```diff
@@ -1,19 +1,12 @@
...
 /administration/installing/custom/automated-installer/system-requirements/index.html /docs/latest/administration/installing/custom/system-requirements/
 /administration/installing/custom/cli/index.html /docs/latest/administration/installing/custom/cli/
 /administration/installing/custom/gui/index.html /docs/latest/administration/installing/custom/gui/
 /administration/installing/index.html /docs/latest/administration/installing/
-/administration/opt-out/index.html /docs/latest/administration/opt-out/
+/administration/opt-out /docs/latest/administration/opt-out/
 /administration/security-and-authentication/index.html /docs/latest/overview/security/
 /administration/service-discovery/index.html /docs/latest/overview/service-discovery/
-/advanced-installer/create-a-script-for-ip-address-discovery/index.html /docs/latest/administration/installing/custom/advanced/
+/advanced-installer/create-a-script-for-ip-address-discovery/index.html /administration/installing/custom/advanced/index.html
```
Comment (karlkfi) [inline redirects]: I think this line was right before...

**Pull request closed without merging**